pkg/standalone-utils/contracts/ProtocolFeeSweeper.sol

```diff
@@ -85,6 +108,13 @@ contract ProtocolFeeSweeper is IProtocolFeeSweeper, SingletonAuthentication, Ree
                 if (feeToken == targetToken) {
                     _transferFeeToken(pool, feeToken, withdrawnBalance);
```

Comment:
maybe require `shouldUnwrap == false` in this case?

It's an edge case, but `shouldUnwrap` doesn't make sense in this context, and it can happen in practice.

Reply:
I think we set the `targetToken` ourselves, and if we want to prevent its use with ERC4626 in this case, we should add a check for `targetToken` inside `setTargetToken`. WDYT?

Reply:
It doesn't really hurt anything though, does it? Worst case you'd just have to unwrap it manually at the destination. There could be a case where you just want the wrapped token directly, so I don't think we should categorically prohibit ERC4626 as targets.